### DISCUSSION TITLE
Remotely connected BMS via esphome

### DISCUSSION THREAD

**Discussion by 20after4** [mr-manuel/venus-os_dbus-serialbattery#64] [Ideas]
I have multiple JBD BMS boards set up and monitored by ESPHome nodes currently, using https://github.com/syssi/esphome-jbd-bms  to interface with the BMS.  I'm thinking about adapting dbus-serialbattery to communicate with home assistant or directly with the esphome node. This is mostly so that I don't have to connect the victron GX directly to the batteries. My batteries are outside while my inverter and other equipment is in my basement.

Does anyone else have any interest in doing this?  I'll definitely make a PR and share my solution if I get it working, just mostly looking for feedback about whether this is even a good idea.

**Comment by mr-manuel** [comment]
Check for dbus-mqtt-battery in my repository. You could use that.
